### DISCUSSION TITLE
"Support AFFiNE, the Open-Source Note-taking tool Sponsored by GitHub! 🚀"

### DISCUSSION THREAD

**Discussion by JackiePengp** [toeverything/AFFiNE#1811] [General]
🎉🎉🎉 Exciting news! AFFiNE, the open-source project for enhancing user experience in text editing and manipulation, has been sponsored by GitHub! 🎉🎉🎉

This is a major milestone moment for the AFFiNE team, and we are grateful for the opportunity to continue developing AFFiNE and bringing new features and improvements to our users.

If you want to support AFFiNE too, you can now make a monthly donation through GitHub Sponsor at different levels, each with its own set of benefits. By becoming a sponsor, you'll be helping us cover our development costs, and expand the capabilities of AFFiNE. 💪

To support AFFiNE, head over to our GitHub Sponsor page: 🔗 https://github.com/sponsors/toeverything

Thank you for considering supporting this amazing project! 🙌

**Comment by JackiePengp** [comment]
if anyone has anything to share or any question to ask, please feel completely free to message me!! I'll be there right away !!

**Reply by fallsdevil** [reply]
Hey, I would like to know if you will have the option to save whiteboard as an image (with a transparent background), because that's only what I need to migrate once and for all, I currently use online versions of Excalidraw and TLdraw for that.
